IntelliTect.TestTools.WindowsTestWrapper/IntelliTect.TestTools.WindowsTestWrapper/DesktopControls.cs

```diff
@@ -145,6 +145,17 @@ protected T FindWinControlByControlId<T>(string controlId, Func<UITestControl, T
             return found;
         }
 
+        protected T FindControlBySibling<T>(Func<UITestControl, T> controlType, UITestControl siblingControl ) where T : UITestControl
+        {
+            if (controlType == null)
+                throw new ArgumentNullException(nameof(controlType));
+            if (siblingControl == null)
+                throw new ArgumentNullException(nameof(siblingControl));
+            T found = controlType(siblingControl);
+            found.SearchConfigurations.Add(SearchConfiguration.NextSibling);
```

Comment:
Oh wait, nevermind... I looked back through some old code and forgot that the NextSibling configuration is weird and totally does work this way (it wasn't just a fluke.)

Added a summary but otherwise I think assuming controlType never returns null is safe in this case.